packages/console/src/utilities/validator.ts

```diff
@@ -16,3 +16,13 @@ export const uriOriginValidator = (value: string) => {
     return false;
   }
 };
+
+export const jsonValidator = (value: string) => {
```

Comment:
maybe we can use `trySafe` from essentials>

Reply:
It seems not easy to return `true` or `false` by using `trySafe`, given that `try...catch` is used in many validators, I created an issue to track. (LOG-5003)